app/classifier/index.cjsx

```diff
@@ -655,6 +672,8 @@ module.exports = React.createClass
       <Classifier {...@props}
         workflow={@props.workflow}
         subject={@state.subject}
+        guide={@state.guide}
```

Comment:
Agreed. Do you think it is better to request the field guide data in the classifier? Or, is it better to move the request back to the PotentialFieldGuide component?